lib/class-wp-rest-global-styles-controller-gutenberg.php

```diff
@@ -0,0 +1,742 @@
+<?php
+/**
+ * REST API: Try: bundle WP_Theme_JSON class instead of inheriting per WordPress version class
+ *
+ * Changes to this class should be synced to the corresponding class
+ * in WordPress core: src/wp-includes/rest-api/endpoints/class-wp-rest-global-styles-controller.php.
+ *
+ * @package    gutenberg
+ * @subpackage REST_API
+ * @since 5.9.0
+ */
+
+if ( class_exists( 'WP_REST_Global_Styles_Controller_Gutenberg' ) ) {
```

Comment:
Why would we need to check for this? Is there a different place where we expect this class to be declared other than this file?

Reply:
No reason. I was just copying what we do in [WP_Theme_Json_Gutenberg](https://github.com/WordPress/gutenberg/blob/111c7e05f6a2f9e9dddf6f8b8b534fe35caaaa5f/lib/class-wp-theme-json-gutenberg.php#L9-L9).

I assumed it was just protocol or something 😄

Reply:
It actually may be "protocol" now? https://github.com/WordPress/gutenberg/pull/52696

Reply:
> It actually may be "protocol" now? https://github.com/WordPress/gutenberg/pull/52696

That'd be it 😄